packages/sanity/src/core/config/types.ts

```diff
@@ -141,6 +141,11 @@ export interface Tool<Options = any> {
    */
   title: string
 
+  /**
+   * Determines whether the tool will control the document title.
```

Comment:
very tiiiiny nit: but for clarity maybe write `document.title`? "Document" is such an overloaded word in sanity 😄 

```suggestion
   * Determines whether the tool will control `document.title`.
```

Reply:
sure!! Thanks!
Tiny things can have a big impact later

Reply:
Very true :)